### PR TITLE
[#2874] improvement(CI): remove SSH service from Hive container

### PR DESCRIPTION
**Pull request by mchades**
### What changes were proposed in this pull request?

 - Remove SSH service from the startup script.
 - Use `hadoop-daemon.sh` to start HDFS services.

### Why are the changes needed?

Fix: #2874 

### Does this PR introduce _any_ user-facing change?

no

### How was this patch tested?

CI pass

### CODE REVIEW

**Comment by jerryshao** [comment]
The change looks good to me. @xunliu do you want take another look?

File: dev/docker/hive/start.sh [hunk 1]
```diff
@@ -4,15 +4,16 @@
 # This software is licensed under the Apache License version 2.
 #
 
-# start ssh
```
Comment (xunliu) [inline dev/docker/hive/start.sh:7]: @mchades  Do we need also remove `apt-get install openssh-server` from `hive/Dockerfile/?

Comment (mchades) [reply dev/docker/hive/start.sh:7]: I'm not sure if removing this service will cause other issues, this requires more testing. 

I suggest keeping it for now and handling it in a separate PR if necessary.

**Review by xunliu — APPROVED** [review]
We can improve Dockerfile in next PR.

**Pull request merged**